### ISSUE TITLE
Datepicker moves if it is to close to the edge

### ISSUE THREAD

**Issue by roxy-dao** [msupply-foundation/open-msupply#2926]
## What went wrong? 😲
Datepicker moves if it is to close to edge, so you can't select a date...... Popping this here since might be able to fix this with: https://mui.com/x/react-date-pickers/custom-components/#popper

https://github.com/msupply-foundation/open-msupply/assets/61820074/125fc709-e47b-4ccc-9fef-8ff3eacd82c7

## How to Reproduce 🔨

Steps to reproduce the behaviour:

## Your environment 🌱

 - Version: 
 - Platform:
   - desktop (macOS)

**Comment by regotaina** [comment]
Error still persists. Cannot select the last quarter of the year (oct/nov/dec) if calendar is on edge of the screen, during a stocktake with multiple batches. 
Uploaded a video on Gdrive

**Comment by roxy-dao** [comment]
Uploading Screen Recording 2024-05-08 at 13.17.13.mov…

I tested on RC branch and seems fine. Unsure what it looks like for you since your video isn't there =[

**Comment by Chris-Petty** [comment]
@regotaina please link the video!

**Comment by regotaina** [comment]
> @regotaina please link the video!
:
[here](https://drive.google.com/file/d/1tg4c0QHF00hQOGJrujjWOy1HZaaysEae/view?usp=drive_link)

**Comment by roxy-dao** [comment]
Sorry just for clarification, the video looks like it's from desktop, correct?

I just tested with an Android right now and seems to be working fine on that too D;

**Comment by regotaina** [comment]
> Member

yeah, desktop app! it collapses if there's no option to scroll down

![Screenshot 2024-05-09 at 10 25 38 AM](https://github.com/msupply-foundation/open-msupply/assets/132425096/78f8b1d6-dedc-491d-8fec-add004933896)

**Comment by roxy-dao** [comment]
Okay can confirm it looks different on windows for web and desktop app 👀
